src/camera_info_manager/zoom_camera_info_manager.py

```diff
@@ -197,7 +209,23 @@ def _update_camera_info(self):
 
 
 class InterpolatingZoomCameraInfoManager(ZoomCameraInfoManager):
-    """A zoom camera info manager that interpolates between several calibrations taken at different zoom levels."""
+    """
+    A zoom camera info manager that interpolates between several calibrations taken at different zoom levels.
+    
+    The calibration results can be as accurate as you need (you increase accuracy by adding more calibration
+    files). E.g. if the camera has only a few discrete zoom steps, you can just calibrate all of them and reach
+    the lowest calibration error possible.
+    
+    Hints for calibration:
+    Use the standard camera calibration package, set a fixed zoom level and perform the calibration. Copy the
+    calibration YAML file to some other location, change the zoom level, and calibrate again. Rename all the 
+    calibration YAML files, so that their names correspond to the calibration_url_template you pass to the
+    constructor.
+    It is advised to always perform calibration on (at least) the highest and lowest zoom step.
+    As the very last, you should run the calibration procedure at the most used zoom level, so that the system
```

Comment:
Definitely not "at the very least"... I meant that the last calibration the user does should be on the most used zoom level (so that the YAML stored in the default location corresponds to the most used zoom level).